CHANGELOG.rst

```diff
@@ -0,0 +1,22 @@
+# Changelog
+All notable changes to this project will be documented in this file.
+
+The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/),
+and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0.html).
+
+## [1.2.0] - Unreleased
+### Added
+- This `CHANGELOG.rst` file.
+- Citation information in the `README` file (see PR #472).
+- Contribution guide on the project's wiki (https://github.com/ExoCTK/exoctk/wiki/Git-Workflow).
+- Procedures for ExoCTK release (https://github.com/ExoCTK/exoctk/wiki/Release-Procedure).
+- Support for installation via PyPI (see PR #385).
+- JWST instrument throughputs for limb-darkening calculations (see PR #458).
+
+### Removed
+- Dropped support for `python3.6` (see PR #471). 
```

Comment:
```suggestion
- Dropped support for Python 3.6 (see PR #471). 
```

Reply:
I think this can just be in plain text, unless 'python3.6' is some sort of keyword or filename that I am missing?